### DISCUSSION TITLE
Rotate around the x/y/z axis while keeping its angle?

### DISCUSSION THREAD

**Discussion by doot56** [KyryloKuzyk/PrimeTween#93] [FAQ] [answered]
```c#
// Rotate 'transform' around the y-axis by 360 degrees in 1 second 
Tween.EulerAngles(transform, startValue: Vector3.zero, endValue: new Vector3(0, 360), duration: 1);
``` 

I tried with a cube ``` Tween.EulerAngles(transform, startValue: new Vector3(-20, 0), endValue: new Vector3(-20, 360), duration: 1); ``` 
 But the whole thing starts rotating. I want to rotate only the y while its rotating and keeping its angle so I can see the top of the cube the whole time kind of like facing me.

**Comment by KyryloKuzyk** [comment, marked as answer]
Try this:
1. Create a new empty object with rotation (20,0). 
2. Make this cube a child of that empty object. 
3. Rotate the cube with Tween.LocalEulerAngles().
